### DISCUSSION TITLE
support for mac os with M1/M2 CPU

### DISCUSSION THREAD

**Discussion by forrestjgq** [debauchee/barrier#1758] [Q&A]
I wonder if there is any plan to release mac os package with M1/M2 CPU support?

or, if not, how to compile one?

**Comment by guitarrensolo** [comment]
Isn't M1 supported yet? And here I am, going crazy trying to figure out why it doesn't work. lol.  

-following-
:)

**Comment by guitarrensolo** [comment]
btw hoping somebody can help me, although I wouldn't know wether it's possible to get any helpful infos out of these screenshots.

![Master](https://user-images.githubusercontent.com/115743871/195710375-79ec1316-62f6-4ee5-a7f5-bee37547196c.png)



![Slave](https://user-images.githubusercontent.com/115743871/195710326-4aeb4d4f-68c1-4205-953e-6409ec876ba0.png)
![Master](https://user-images.githubusercontent.com/115743871/195710333-c11e948a-0f3a-44e7-9172-d2c36e8883ba.png)

**Reply by guitarrensolo** [reply]
Master is M1, slave i7
